packages/backend/src/queue/processors/CleanRemoteNotesProcessorService.ts

```diff
@@ -155,12 +162,12 @@ export class CleanRemoteNotesProcessorService {
 		// | fff | fff    | TRUE        |
 		// | ggg | ggg    | FALSE       |
 		//
-		const candidateNotesQuery = this.db.createQueryBuilder()
+		const candidateNotesQuery = ({ limit }: { limit: number }) => this.db.createQueryBuilder()
```

Comment:
My bad, I think the primary issue is here. The adaptive limit was not being picked up causing the timeout. 

But it should be much more robust now.